src/bin/server.ts

```diff
@@ -46,16 +46,67 @@ interface Args {
   if (configPath) {
     try {
       options = require(configPath);
-      // make sure that networkKey is passed as buffer.
-      // accept both zwave2mqtt format as ozw format
-      if (options.networkKey && options.networkKey.length === 32) {
-        options.networkKey = Buffer.from(options.networkKey, "hex");
+      const securityKeyNames = [
+        "S0_Legacy",
+        "S2_AccessControl",
+        "S2_Authenticated",
+        "S2_Unauthenticated",
+      ];
+      // We prefer the securityKeys option over the networkKey one
+      for (const key of securityKeyNames) {
+        if (
+          options.securityKeys &&
+          options.securityKeys[key] &&
+          options.securityKeys[key].length === 32
+        ) {
+          options.securityKeys[key] = Buffer.from(
+            options.securityKeys[key],
+            "hex"
+          );
+        } else if (
+          options.securityKeys &&
+          options.securityKeys[key] &&
+          options.securityKeys[key].includes("0x")
```

Comment:
should this be startsWith?

Reply:
This check is to convert OZW keys to Z-Wave JS format. OZW never supported S2, do you really need to support its key format here? Users should be encouraged to drop OZW key formats, IMO.

Reply:
I have dropped support for OZW formats for the S2 keys